pages/about.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <CContainer>
+    <h1>About</h1>
+    <h2>The DAEDALUS Explore dashboard</h2>
+    <p>
+      DAEDALUS Explore is an interactive dashboard tool that allows users to
+      simulate the health, education and economic impacts of hypothetical future
+      pandemic scenarios for {{ numberOfPandemics }} pandemics of varying severity in {{ numberOfCountries }}
+      countries. The {{ numberOfPandemics }} pandemics are informed by the characteristics of
+      pathogens that have caused historic pandemics.
+    </p>
+    <p>The dashboard provides estimates of:</p>
+    <ul>
+      <li v-for="estimate in appStore.metadata?.results.time_series" :key="estimate.id">
+        {{ estimate.label }}
+      </li>
+      <li>Economic losses</li>
+    </ul>
+    <p>
+      DAEDALUS Explore allows users to choose a country, a pandemic, and a
+      response amongst four options (no closures, school closures, business
+      closures, and elimination). Response is the mitigation that policy makers
+      can choose once the pandemic strikes. DAEDALUS Explore allows users to
+      explore the inherent trade-off between different response strategies. For
+      example, business closures may result in higher economic costs, but lower
+      deaths compared to a response that does not mandate closures.
+    </p>
+    <p>
+      The losses associated with pandemics can be influenced by investments into
+      global vaccine R&D, manufacturing and hospital capacity. Users can
+      explore the impact of such investments on health, education and economic
+      pandemic impacts by comparing the outcomes of scenarios with different
+      investments. The impacts of all pandemic scenarios for the countries are
+      projected in terms of GDP loss (absolute, and in % of pre-pandemic GDP),
+      the long-term income loss caused by the interruption of education in schools,
+      and in terms of lost life years, valued in monetary terms.
+    </p>
+
+    <h2>The DAEDALUS model</h2>
+    <p>
+      DAEDALUS Explore is generated by the integrated economic-epidemiological
+      model DAEDALUS and uses publicly accessible data sources. For full details
+      on the underlying DAEDALUS model visit
+      <a href="https://github.com/jameel-institute/daedalus" target="_blank">Github</a>
```

Comment:
I noticed that Github styles itself camelcase: GitHub.
```suggestion
      <a href="https://github.com/jameel-institute/daedalus" target="_blank">GitHub</a>
```